src/localisation/string_ids.h

```diff
@@ -2453,6 +2453,52 @@ enum {
 	STR_CHEAT_DISABLE_TRAIN_LENGTH_LIMIT_TIP = 5811,
 	
 	STR_SHORTCUT_SHOW_MULTIPLAYER = 5812,
+	
+	STR_HARDWARE_DISPLAY_TIP                  = 5813,
```

Comment:
I'll switch back to spaces for now. I think the vertical alignment may be a job for another commit